corehq/apps/users/templates/users/edit_commcare_user.html

```diff
@@ -189,46 +187,51 @@ <h4>[Support only] Assigned Locations</h4>
         </div>
       </div>
     {% endif %}
-    {% if request|toggle_enabled:"SUPPORT" %}
-      <div class="tab-pane" id="user-support">
-        <div class="panel-body">
-          <table class="table table-striped table-hover">
-            <tbody>
-              <tr>
-                <td>Created On</td>
-                <td>{{ couch_user.created_on }}</td>
-              </tr>
-              <tr>
-                <td>Last Modified</td>
-                <td>{{ couch_user.last_modified }}</td>
-              </tr>
-              <tr>
-                <td>Last Login</td>
-                <td>{{ couch_user.last_login }}</td>
-              </tr>
-              <tr>
-                <td>Devices</td>
-                <td>
-                  <ul class="list-unstyled">
-                    {% for device in couch_user.devices %}
-                      <li>{{ device.last_used }} - {{ device.device_id }}</li>
-                    {% endfor %}
-                  </ul>
-                </td>
-              </tr>
-              <tr>
-                <td>Last Sync</td>
-                <td>{{ couch_user.reporting_metadata.last_sync_for_user.sync_date }}</td>
-              </tr>
-              <tr>
-                <td>Last Form Submission</td>
-                <td>{{ couch_user.reporting_metadata.last_submission_for_user.submission_date }}</td>
-              </tr>
-            </tbody>
-          </table>
-        </div>
+    <div class="tab-pane" id="user-metadata">
+      <div class="panel-body">
+        <table class="table table-striped table-hover">
+          <tbody>
+            <tr>
+              <td>{% trans "Created On" %}</td>
+              <td>{{ couch_user.created_on|to_user_time:request }}</td>
+            </tr>
+            <tr>
+              <td>{% trans "Last Modified" %}</td>
+              <td>{{ couch_user.last_modified|to_user_time:request }}</td>
+            </tr>
+            <tr>
+              <td>{% trans "Last Login" %}</td>
+              <td>{{ couch_user.last_login|to_user_time:request }}</td>
+            </tr>
+            <tr>
+              <td title="Approximate date/time of the last time this user synced with the server from CommCare">
```

Comment:
The `titles` are informative, but hard to discover, how about switching them to help bubbles? Example: https://github.com/dimagi/commcare-hq/blob/e7c92eb5c7a2fe53d01f97e43b4e7286a18a0460/corehq/apps/users/templates/users/roles_and_permissions.html#L71-L79

Reply:
Oops, and I forgot to translate them :facepalm:  - apparently I'm bad at that.